projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -17,12 +17,11 @@ import getMediaToken from '../../../../../lib/get-media-token/index.native';
 import { getVideoPressUrl } from '../../../../../lib/url';
 import { usePreview } from '../../../../hooks/use-preview';
 import addTokenIntoIframeSource from '../../../../utils/add-token-iframe-source';
+import { VideoPressIcon } from '../icons';
 import style from './style.scss';
 
 const VIDEO_PREVIEW_ATTEMPTS_LIMIT = 10;
-
-// The preview is ready when it has a height property
-const isPreviewReady = preview => !! preview?.height;
+const DEFAULT_PLAYER_ASPECT_RATIO =  16 / 9; // This is the observed default aspect ratio from VideoPress embeds.
```

Comment:
I found the `16:9` aspect ratio matches the default embed aspect ratio. I did find [this reference](https://github.com/Automattic/jetpack/blame/rnmobile/update/vp-loading-screen/projects/packages/videopress/src/class-block-editor-content.php#L108) to the `16:9` aspect ratio in the VP package but I don't think that's what's served to the editor.  Either way I found this is working well